apps/end-tenancy/translations/src/en/pages.json

```diff
@@ -48,7 +48,8 @@
   "tenant-details-name": {
     "header": {
       "what": {
-        "check": "What's the full name of the {{next}}person that you’d like to check?"
+        "check": "What's the full name of the {{next}}person that you’d like to check?",
+        "request": "What's the name of the {{next}}person you'd like to request a Notice of Letting to a Disqualified Person for?"
```

Comment:
templates have access to `res.locals` so we can modify content depending on certain situations. In this case {{next}} is an empty string if you haven't added a tenant, but it is `"next "` if you have. This is in locals in the controller